src/types.ts

```diff
@@ -15,7 +15,7 @@ export type ItemResponse = {
 
 export enum ItemType {
   DocumentType='DocumentType',
-  CollectionType='CollectionTyp',
+  CollectionType='CollectionType',
```

Comment:
fix typo, folder doesn't show up otherwise

Reply:
Thanks for pointing it out!